osquery/requests/serializers/json.h

```diff
@@ -0,0 +1,56 @@
+/*
+ *  Copyright (c) 2014, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#pragma once
+
+#include "osquery/requests/requests.h"
+
+namespace osquery {
+
+/**
+ * @brief JSON Serializer
+ */
+class JSONSerializer : public Serializer {
```

Comment:
You can keep the JSON serializer implementation along side the remote requests but the various serialization/deserialization (read: underlying boost API calls) methods we often repeat should be organized as a dependency task. :P Truthfully, it is way past time to reorganize these methods, however you decide will be great!